school/models.py

```diff
@@ -0,0 +1,84 @@
+import datetime
```

Comment:
These models need a lot of love. What other fields do we need to store for each of these?